LibraryWebApp/Models/ReservationCart.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace LibraryWebApp.Models
 {
-    public class ShoppingCart
+    public class ReservationCart
```

Comment:
There should be a "ReservationCartEntry", because it means that you can only add one book to reservation or you have multiple reservation carts when you get it from session.